File: src/main/java/com/sfh/pokeRogueBot/model/modifier/impl/AddVoucherModifierItem.java
package com.sfh.pokeRogueBot.model.modifier.impl;

import com.sfh.pokeRogueBot.model.enums.VoucherType;
import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper = true)
public class AddVoucherModifierItem  extends ModifierItem {

    public static final String TARGET = "AddVoucherModifierType";

    private VoucherType vouchertype;
    private int count;

}

File: src/main/kotlin/com/sfh/pokeRogueBot/model/modifier/impl/AddVoucherModifierItem.kt
package com.sfh.pokeRogueBot.model.modifier.impl

import com.sfh.pokeRogueBot.model.enums.VoucherType

class AddVoucherModifierItem : ModifierItem() {

    companion object {
        const val TARGET = "AddVoucherModifierType"
    }

    var vouchertype: VoucherType? = null
    var count: Int = 0
}